### PR TITLE
Update connectedmachine extension

### PR DESCRIPTION
**Pull request by TylerLeonhardt**
---

This updates the connectedmachine extension to 2020-08-02 of the spec. In doing this I also changed the autogenerated tests to test more useful things.

> I was told to move the changed tests to the custom folder but after doing that then the test didn't run... so I'm not sure what to do there...

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [x] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by TylerLeonhardt** [comment]
```
-  FAIL - HIGH severity: option_length_too_long
    Parameter: connectedmachine machine-extension create, `auto_upgrade_minor_version` - The lengths of all options ['--auto-upgrade-minor-version'] are longer than threshold 22. Argument auto_upgrade_minor_version must have a short abbreviation.
    Parameter: connectedmachine machine-extension update, `auto_upgrade_minor_version` - The lengths of all options ['--auto-upgrade-minor-version'] are longer than threshold 22. Argument auto_upgrade_minor_version must have a short abbreviation.
```

Is there a way to ignore this? I don't really think it's a parameter that can be shortened.

Also, the Integration Tests don't appear to be connectedmachine's fault so I guess I should ignore those failures?

**Comment by yungezz** [comment]
hi @fengzhou-msft could you pls help to review the PR for Azure Arc  for Servers? thanks

**Comment by TylerLeonhardt** [comment]
This PR should fix the lint issue:
https://github.com/Azure/azure-rest-api-specs/pull/10789

File: src/connectedmachine/report.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
+# Azure CLI Module Creation Report
```
Comment (yungezz) [inline src/connectedmachine/report.md:1]: what's this for? It should be auto-gen code report, no need to checkin

File: src/connectedmachine/azext_connectedmachine/vendored_sdks/connectedmachine/py.typed [hunk 2]
```diff
@@ -0,0 +1 @@
+# Marker file for PEP 561.
```
Comment (yungezz) [inline src/connectedmachine/azext_connectedmachine/vendored_sdks/connectedmachine/py.typed:1]: what's this file used for?

File: src/connectedmachine/azext_connectedmachine/tests/latest/test_connectedmachine_scenario_coverage.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+|Scenario|Result|ErrorMessage|ErrorStack|ErrorNormalized|StartDt|EndDt|
```
Comment (yungezz) [inline src/connectedmachine/azext_connectedmachine/tests/latest/test_connectedmachine_scenario_coverage.md:1]: this  no need to be checked in also.

**Comment by TylerLeonhardt** [comment]
Closing in favor of https://github.com/Azure/azure-cli-extensions/pull/2364

File: src/connectedmachine/azext_connectedmachine/vendored_sdks/connectedmachine/py.typed (hunk 2 above)
Comment (TylerLeonhardt) [reply src/connectedmachine/azext_connectedmachine/vendored_sdks/connectedmachine/py.typed:1]: No idea it was there as a part of the generation

File: src/connectedmachine/report.md (hunk 1 above)
Comment (TylerLeonhardt) [reply src/connectedmachine/report.md:1]: No idea it was there as a part of the generation

**Pull request closed without merging**